modules/eks/variables.tf

```diff
@@ -45,8 +45,9 @@ variable "iam_account_id" {
 
 variable "iam_role_mapping" {
   type = list(object({
-    name = string
-    arn  = string
+    name   = string
```

Comment:
Maybe these could be "k8s_role_name", "iam_role_arn", "k8s_groups" to make it more clear?